ajax/libs/coolprop/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "coolprop",
+  "description": "CoolProp is a thermophysical property database. It offers similar functionality to REFPROP",
+  "version": "6.3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CoolProp/CoolProp.git"
+  },
+  "keywords": [
+    "coolprop",
+    "refprop",
+    "thermophysical"
+  ],
+  "author": "ibell, jowr, wandadars, etc.",
+  "license": "MIT",
+  "homepage": "https://github.com/CoolProp/CoolProp/",
+  "filename": "coolprop.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/CoolProp/CoolProp.git",
+    "fileMap": [
+      {
+        "basePath": "wrappers/Javascript",
+        "files": [
+          "coolprop.js",
+          "coolprop.min.js",
+          "coolprop.wasm"
+        ]
```

Comment:
These files do not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript